rebasehelper/specfile.py

```diff
@@ -136,7 +142,11 @@ def _update_data(self):
         :return: 
         """
         # Load rpm information
-        self.spc = rpm.spec(self.path)
+        try:
+            self.spc = rpm.spec(self.path)
+        except ValueError:
+            message = "Problem with parsing SPEC file '%s'" % self.path
+            raise SpecParseError(message)
```

Comment:
I think you don't need to define new type of Exception and raise it here. Instead of this just raise RebaseHelperError directly here. Also you can format the message in the constructor of the Exception - no need to define variable. I don't remember what is used in other parts of rebase-helper, but you can use " ".format() syntax for formating of strings.